user-docs/pupil-capture.md

```diff
@@ -183,6 +183,27 @@ This method is for special situations and far distances. Usually not required.
 1. Press `c` on your keyboard or click the blue circular `C` button in the left hand side of the world window to stop calibration.
 
 
+#### Fingertip Calibration
+
+Calibrate using your fingertip! Move your index finger into your field of view while looking at the fingertip. 
+
+This is a two step process: 
+(1) calibrating for skin tone of the participant
+(2) collecting calibration samples.
+
+1. Select Fingertip Calibration
+1. Press `c` on your keyboard or click the blue circular `C` button on the left hand side of the world window to start calibration.
+1. Cover the rectangles on the left side of the window with your index finger
```

Comment:
@ChingT - which side of the hand should you be looking at? Example - palm of hand facing you, or back of hand facing you/world camera?